copyGcno.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+if [ ! -d "$2" ]; then
+  echo "$2 does not exist."
+  exit 1
+fi
+
+builddirPrefix=$1
+gcnoFiles=`ls -LR  ${builddirPrefix}`
+gcnoSuffix="gcno"
+dirSuffix=":"
+currentDir=""
+
+
+for eachfile in $gcnoFiles
+do
+    if [[ $eachfile == *$dirSuffix ]]; then
+    currentDir=$eachfile
+    fi
+    if [[ $eachfile == *$gcnoSuffix ]]; then
+    sudo cp  ${currentDir%?}/$eachfile $2
+    fi
+done
+
```

Comment:
It seems too complex, I suggest you:

1. Use existing `MESON*` environment variables that meson exposes during install script execution, you can display them by running `set > /tmp/meson-env.txt` and checking the file
2. You could you single line combination of find and cp commands